src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -246,7 +246,12 @@ export default [
             Clipboard.setString(selection.replace('mailto:', ''));
             hideContextMenu(true, ReportActionComposeFocusManager.focus);
         },
-        getDescription: (selection) => selection.replace('mailto:', ''),
+        getDescription: (selection) =>
```

Comment:
This can get out of hand pretty quickly - you can consider extracting these functionalities into 2 pure functions like `trimMailTo` and `trimZeroWidthSpace` and describe them well for the others.

Think about yourself stumbling upon this 6 months from now 😉

Otherwise this looks good to me!

Reply:
Thanks, resolved in 81a50658949868f7565831f9dc2a1e0a591c9fda